app/components/Settings/Fiat.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import FaAngleLeft from 'react-icons/lib/fa/angle-left'
+import Isvg from 'react-inlinesvg'
+import checkIcon from 'icons/check.svg'
+import styles from './Fiat.scss'
+
+const Fiat = ({ fiatTicker, fiatTickers, disableSubMenu, setFiatTicker }) => (
+  <div>
+    <header className={styles.submenuHeader} onClick={disableSubMenu}>
```

Comment:
Could we make the menu close when you click outside of it too, not just on the alias? The UI feels strange when the menu stays open even as you navigate around other parts of the app.

Reply:
Added in latest commit